### PR TITLE
add cronjob to downscale daemonsets

### PR DESCRIPTION
**Pull request by QuantumEnigmaa**
This PR adds cronjobs able to scale down/up daemonsets during downtime.

### CODE REVIEW

**Comment by QuantumEnigmaa** [comment]
I tested both cronjobs as simple jobs on a MC and it works perfectly.

File: helm/kube-downscaler/values.yaml [hunk 1]
```diff
@@ -18,6 +18,23 @@ interval: 60
...
+    downscaleSchedule: "0 2 * * 6"
+    # The upscaleSchedule Field must match the scalingInfo field's ending time
+    upscaleSchedule: "0 22 * * 7"
+  # A list of the daemonsets that should be downscaled. Each list's member must be as follows :
+  # - name: <daemonset-name>
+  #   namespace: <daemonset-namespace>
+  list: []
+
 scalingInfo: |
   DEFAULT_DOWNTIME="Sat-Sat 02:00-24:00 CET,Sun-Sun 00:00-22:00"
```
Comment (pipo02mix) [inline helm/kube-downscaler/values.yaml:39]: is this just for informational purposes?

**Review by pipo02mix — APPROVED** [review]
LGTM I was doubting between a single downscaler job for all daemonset or yours that creates a cronjob per daemonset. Both have pros and cons, so let's go with yours :)

File: helm/kube-downscaler/values.yaml (hunk 1 above)
Comment (QuantumEnigmaa) [reply helm/kube-downscaler/values.yaml:39]: Yes as a user config should be provided for each MC with its own list of daemonset to downscale.

**Comment by QuantumEnigmaa** [comment]
> I was doubting between a single downscaler job for all daemonset or yours that creates a cronjob per daemonset. Both have pros and cons,

Well having one cronjob per daemonset is way easier to set in my opinion

**Pull request merged**